draft-ietf-rats-corim.md

```diff
@@ -1258,26 +1258,25 @@ There are scope-related questions that need to be answered.  ([^tracked-at] http
 
 ## Extensibility {#sec-extensibility}
 
-The base CORIM schema is described using CDDL {{-cddl}} that can be extended
-only at specific allowed points known as "extension points"
+The base CoRIM document definition is described using CDDL {{-cddl}} that can be extended only at specific allowed points known as "extension points".
 
-The following types of extensions are supported in CoRIM
+The following types of extensions are supported in CoRIM:
 
 ### Map Extensions
-Map Extensions provides extensibility support to CoRIM Map structures.
-CDDL map extensibility enables a CoRIM profile to extend the base CoRIM definition.
-CDDL map extension points have the form `($$NAME-extension)` where "NAME" is the name of the map
-and '$$' signifies map extensibility. Typically, map extension requires a convention
-for code point naming that avoids code-point reuse.
+
+Map extensions provide extensibility support to CoRIM map structures.
+CDDL map extensibility enables a CoRIM profile to extend the base CoRIM CDDL definition.
+CDDL map extension points have the form `($$NAME-extension)` where "NAME" is the name of the map and '$$' signifies map extensibility.
+Typically, map extension requires a convention for code point naming that avoids code-point reuse.
 Well-known code points may be in a registry, such as CoSWID {{-coswid-reg}}.
 Non-negative integers are reserved for IANA to assign meaning globally.
 
 ### Data Type Extensions
 
-Data type extensibility has the form `($NAME-type-choice)` where "NAME" is the type name
-and '$' signifies type extensibility.
+Data type extensibility has the form `($NAME-type-choice)` where "NAME" is the type name and '$' signifies type extensibility.
 
-Schema extensions (Map or Data Type) should be documented to facilitate interoperability. CoRIM profiles are best used to document vendor or industry defined extensions.
+New data type extensions MUST be documented to facilitate interoperability.
```

Comment:
This okay? Introducing normative language might be out-of-scope of the issue addressed.

Reply:
No, CoRIM profiles can be entirely private and not be meant for use outside of a closed circuit.